src/TypeDefinition.js

```diff
@@ -381,6 +381,7 @@ export type NavigationTransitionProps = {
   // is the index of the scene
   scene: NavigationScene,
   index: number,
+  navigation: NavigationScreenProp<NavigationRoute, NavigationAction>,
```

Comment:
My thought is that hopefully we can use the transitioners' memoization of the transitioner props, instead of `_getChildNavigation`. Do you think that could work?